python/tests/expressions/test_operations.py

```diff
@@ -17,26 +17,26 @@
 
 import pytest
 
-from iceberg import expressions
+from iceberg.expressions import operations
```

Comment:
Shouldn't `operations` go into `base.py` and then get exposed in `__init__.py`? I would probably leave the tests unchanged and use `expressions.Operation.*`.

Reply:
+1 that operations should go into base.py but would it be helpful for the tests to use the full imports? In other words `from iceberg.expressions import base`. It makes it a bit easier to jump from looking at a test to the relevant source code.

Reply:
Thank you, updated per your suggestions!